go.mod

```diff
@@ -145,7 +145,7 @@ replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-	github.com/tendermint/tendermint => github.com/sei-protocol/sei-tendermint v0.1.171
+	github.com/tendermint/tendermint => github.com/sei-protocol/sei-tendermint v0.1.179-0.20230315064114-c903df251edf
```

Comment:
Was this intentional?

Reply:
Can we tag a new tendermint release instead

Reply:
+1. Let's revert this and the protocol-team can cut a new release

Reply:
yes will update it to a tm release after this is merged https://github.com/sei-protocol/sei-tendermint/pull/85